example/src/AccordionExample.js

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { Accordion, withTheme, AccordionItem } from "@draftbit/ui";
+import Section, { Container } from "./Section";
+
+function AccordionExample({ theme }) {
+  return (
+    <Container style={{ backgroundColor: theme.colors.background }}>
+      <Section title="Accordion list with AccordionItem">
+        <Accordion
+          openColor={"primamry"}
+          closedCloser={"secondary"}
+          icon={"folder"}
+          iconSize={26}
+          style={{ fontWeight: "bold" }}
+          title={"First"}
+          children={
```

Comment:
The React api for children looks like this;

```
<Accordion>
	<AccordionItem icon={"insert-drive-file"} title={"First file"} />
</Accordion>
```

Reply:
done